### DISCUSSION TITLE
Looking for an old 32-bit Windows Binary for MITMDUMP.exe

### DISCUSSION THREAD

**Discussion by madeff** [mitmproxy/mitmproxy#6050] [Q&A]
If anyone still has a copy of a 32-bit Windows binary for mitmdump.exe, please reply.  It would save me a lot of time and effort.

Thanks.

**Comment by Prinzhorn** [comment]
It's all here https://mitmproxy.org/downloads/

Edit: it appears you already known that (https://github.com/mitmproxy/mitmproxy/discussions/6049)? So I probably don't know what you're looking for.

**Comment by madeff** [comment]
Seems all the binaries on the download page are 64-bit only. On Apr 5, 2023, at 2:00 AM, Alexander Prinzhorn ***@***.***> wrote:﻿
It's all here https://mitmproxy.org/downloads/
